### ISSUE TITLE
Machine files support

### ISSUE THREAD

**Issue by 0dragosh** [budimanjojo/talhelper#163]
Hi @budimanjojo !

I was trying to [setup tailscale as a Talos extension](https://github.com/siderolabs/extensions/tree/main/network/tailscale) and it needs `machine/files` support, was wondering if you could add that in pretty please?

```
machine:
  install:
    extensions:
      - image: ghcr.io/siderolabs/tailscale:1.44.0
  files:
    - content: |
        TS_AUTHKEY=<your auth key>
      permissions: 0o644
      path: /var/etc/tailscale/auth.env
      op: create
```

**Comment by 0dragosh** [comment]
The env vars that you put in the file are per node, so yes I believe this needs to be per node.

**Comment by budimanjojo** [comment]
@0dragosh I'll release it soon, you can now do it with:
```
nodes:
  extensions:
    - image: image: ghcr.io/siderolabs/tailscale:1.44.0
  machineFiles:
    - content: |
        TS_AUTHKEY=<your auth key>
      permissions: 0o644
      path: /var/etc/tailscale/auth.env
      op: create
```

**Comment by budimanjojo** [comment]
`v1.10.1` released with this addition, it should be `v1.10.0` though but it's too late and I'm lazy to undo release :rofl:

**Comment by 0dragosh** [comment]
Awesome, thank you so much!
